Sources/ClientRuntime/Config/Context+Config.swift

```diff
@@ -0,0 +1,49 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import class Smithy.Context
+import class Smithy.ContextBuilder
+import struct Smithy.AttributeKey
+
+public extension Context {
```

Comment:
Allow the client config object to be placed into the context.  This allows the client config to be passed to the S3 Express identity resolver, so that it can use the S3 client to obtain S3 Express credentials.